src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml

```diff
@@ -10,9 +10,9 @@
             <div class="dropdown-divider"></div>
             @if (await AuthorizationService.AuthorizeAsync(User, Permissions.ManageOwnUserInformation))
             {
-                <a class="dropdown-item" asp-area="OrchardCore.Users" asp-action="Edit" asp-controller="Admin" asp-route-returnUrl="@FullRequestPath"><span><i class="far fa-address-card" aria-hidden="true"></i></span>&nbsp;<span> @T["Profile"]</span></a>
+                <a class="dropdown-item" asp-area="OrchardCore.Users" asp-action="Edit" asp-controller="Admin" asp-route-returnurl="@FullRequestPath"><span><i class="far fa-address-card" aria-hidden="true"></i></span>&nbsp;<span> @T["Profile"]</span></a>
```

Comment:
Please check if this still works and you're correctly redirected to the URL if you run OC via the dotnet CLI, not from VS via IIS Express, since casing can matter.